Cargo.toml

```diff
@@ -26,12 +26,13 @@ soroban-sdk = { path = "soroban-sdk" }
 soroban-auth = { path = "soroban-auth" }
 soroban-spec = { path = "soroban-spec" }
 soroban-sdk-macros = { path = "soroban-sdk-macros" }
-soroban-env-common = { git = "https://github.com/stellar/rs-soroban-env", rev = "b08a54a7" }
-soroban-env-guest = { git = "https://github.com/stellar/rs-soroban-env", rev = "b08a54a7" }
-soroban-env-host = { git = "https://github.com/stellar/rs-soroban-env", rev = "b08a54a7" }
-soroban-env-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "b08a54a7" }
-soroban-native-sdk-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "b08a54a7" }
+soroban-env-common = { git = "https://github.com/stellar/rs-soroban-env", rev = "16402efc" }
+soroban-env-guest = { git = "https://github.com/stellar/rs-soroban-env", rev = "16402efc" }
+soroban-env-host = { git = "https://github.com/stellar/rs-soroban-env", rev = "16402efc" }
+soroban-env-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "16402efc" }
+soroban-native-sdk-macros = { git = "https://github.com/stellar/rs-soroban-env", rev = "16402efc" }
 stellar-xdr = { git = "https://github.com/stellar/rs-stellar-xdr", rev = "fee9a43" }
+wasmi = { package = "soroban-wasmi", git = "https://github.com/stellar/wasmi", rev = "a61b6df" }
```

Comment:
@graydon It looks like the commit of wasmi that soroban-env-host is dependended on in https://github.com/stellar/rs-soroban-env/pull/422 is a commit on a pull request. The commit we have to reference in all our repos will need updating the moment that PR closes because the commit won't be available anymore. Just letting you know so that you know that needs to happen when the final commit is worked out.